Dockerfile

```diff
@@ -17,11 +17,24 @@ RUN poetry install
 RUN curl -sL https://deb.nodesource.com/setup_16.x | bash -
 RUN apt-get install -y nodejs
 
+# Install dependencies and run indexing on the local codebase
+RUN automata install-indexing
+
+# Create a script that will be run when the container is started
+RUN echo "#!/bin/bash\n\
+poetry run automata configure --GITHUB_API_KEY=\$GITHUB_API_KEY --OPENAI_API_KEY=\$OPENAI_API_KEY\n\
+poetry run automata run-code-embedding\n\
+poetry run automata run-doc-embedding --embedding-level=2\n\
+exec \"\$@\"" > entrypoint.sh
+RUN chmod +x entrypoint.sh
+
+# Set this script as the entrypoint for the Docker container
+ENTRYPOINT ["./entrypoint.sh"]
+
 # Instructions on how to run the Docker container
 # These instructions will not be executed during the Docker build
 # They are merely here for the user's convenience
 # Run these commands in the terminal to build and run the Docker container:
-# docker build -t automata .
-# docker run -it --name test automata bash
-# docker stop test
-# docker rm test
+# docker run --name automata_container -it --rm -e OPENAI_API_KEY=<your_openai_key> -e GITHUB_API_KEY=<your_github_key> ghcr.io/emrgnt-cmplxty/automata:latest
```

Comment:
This is the command that will need to be updated in the Readme

Reply:
Totally agree! I will do it during this PR :)

Reply:
And it is done! Good call @NolanTrem!